### PR TITLE
moyo product-card

### PR DESCRIPTION
**Pull request by toshya**
- [DEMO LINK](https://TShmyhol.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://TShmyhol.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/default.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.page {
+  margin: 0;
+
+  font-family: Roboto, sans-serif;
+}
+
+h2,
+p {
+  margin: 0;
+}
```
Comment (Oleksii-Siviolov) [inline src/default.css:10]: I think, it is better to group them all in one, like this:
```
.page,
h1,
p {
  margin: 0;
  font-family: "Roboto", sans-serif;
}
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,64 @@
...
+  </head>
+  <body class="page">
+    <section class="card" data-qa="card">
+      <div class="card__product-photo">
+        <img
+        class="card__product"
+        src="./images/imac.jpeg"
+        alt="iMac"
+      >
+      </div>
```
Comment (Oleksii-Siviolov) [inline src/index.html:28]: If I understand it correct, you don't need to create container for your image, 'cause it is not a BEM block. Image is a child of it.

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,64 @@
...
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+        </div>
+
+        <a class="card__review_total">
+          Reviews: 5
+        </a>
```
Comment (Oleksii-Siviolov) [inline src/index.html:47]: Where do you use `card__review_stars` in your CSS? Div with stars is a block inside other element - no need to refer to 'card'-block in it and also to parent div with class 'card__review' as it is not a block, but element.
`Reviews: 5` is a text - no need to make a link. You could use <p>.

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,64 @@
...
+      </div>
+
+      <div class="card-price">
+        <p class="card-price__price">Price:</p>
+        <p class="card-price__amount">$2,199</p>
+      </div>
```
Comment (Oleksii-Siviolov) [inline src/index.html:53]: I think, it is better that div has class like 'card__price-container' with 'card__price' and 'card__amount' in it.

File: src/stars.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  width: 16px;
+  height: 17px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
```
Comment (Oleksii-Siviolov) [inline src/stars.css:19]: It won't work, if you change to `stars--5` in your HTML. Add last one:
`.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url(./images/star-active.svg);`

File: src/var.css [hunk 6]
```diff
@@ -0,0 +1,8 @@
+.page {
+  --min-font-size: 12px;
+  --min-font-weight: 400;
+  --max-font-weight: 700;
+  --font-dark-blue-color: rgb(6, 11, 53);
+  --font-light-grey-color: rgb(97, 96, 112);
+  --buttton-light-blue: rgb(0, 172, 220);
+}
```
Comment (Oleksii-Siviolov) [inline src/var.css:8]: If I remember correct, variables are claimed with `:root {}`

**Review by Oleksii-Siviolov — CHANGES_REQUESTED** [review]
Hi! Thank you, for trusting me your code, hope I can suggest anything :)

File: src/default.css [hunk 7]
```diff
@@ -0,0 +1,6 @@
+.page,
```
Comment (Olexandr6) [inline src/default.css:1]:
```suggestion
body,
```

File: src/default.css [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+h2,
+p {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
```
Comment (Olexandr6) [inline src/default.css:6]: Better to rename this file to `normalize.css`

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,62 @@
...
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page">
```
Comment (Olexandr6) [inline src/index.html:20]: Why do you need this `page` class?

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,62 @@
...
+        <p class="card__amount">$2,199</p>
+      </div>
+
+      <a
+        href="home"
+        class="card__buy"
+        data-qa="hover"
+      >
+      Buy
+      </a>
```
Comment (Olexandr6) [inline src/index.html:59]:
```suggestion
      >
      	Buy
      </a>
```

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,102 @@
+@import url(./default.css);
+
+@import url(./stars.css);
+
+@import url(./var.css);
```
Comment (Olexandr6) [inline src/style.css:5]: file with vars should be imported in the first queue
```suggestion
@import url(./default.css);
@import url(./var.css);

@import url(./stars.css);

```

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+.card {
```
Comment (Olexandr6) [inline src/style.css:7]: [Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 11th point

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+  margin-bottom: 16px;
+  width: 166px;
+  text-align: left;
+
+  font-size: 10px;
+  color: rgb(97, 96, 112);
+  font-weight: var(--min-font-weight);
+}
+
+.reviews {
```
Comment (Olexandr6) [inline src/style.css:43]: [Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 11th point

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+
+  height: 38px;
+  font-size: 14px;
+  text-transform: uppercase;
+  width: 164px;
+  border-radius: 5px;
+  font-weight: var(--max-font-weight);
+  background-color: var(--buttton-light-blue);
+  color: rgb(255, 255, 255);
+  text-decoration: none;
```
Comment (Olexandr6) [inline src/style.css:93]: Group styles - https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Fit it in all places

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
You marked you checked the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md), but it looks like you haven't.

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**